src/utils/color.js

```diff
@@ -0,0 +1,12 @@
+export default {
```

Comment:
Sorry, should have posted this before but this is what I was thinking: 

<img width="183" alt="Screenshot 2020-03-05 at 10 22 11" src="https://user-images.githubusercontent.com/25960351/75972396-4360a280-5ecb-11ea-8480-d781d8feaaa6.png">

Reply:
Sorry to be annoying ':)